frontend/src/tabs/Account.js

```diff
@@ -1,13 +1,60 @@
-import React from 'react'
-import './Account.css';
-import BasicTab from './BasicTab.js';
+import React, { useState } from "react";
+import "./Account.css";
+import BasicTab from "./BasicTab.js";
 
-const Account = props => {
+const Account = (props) => {
   // TODO: Create the account tab w/ wallet address, username, pixel info, top X % users ( pixels placed? ), ...
+  const [username, setUsername] = useState("");
+  const [isUsernameSaved, saveUsername] = useState(false);
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(event);
+    setUsername(username);
+    saveUsername(true);
+  }
   return (
     <BasicTab title="Account">
+      <div className="Account__flex">
+        <p>Address:</p>
+        <p className="Account__wrap">
+          0x0000000000000000000000000000000000000000000000000000000000000000
+        </p>
+      </div>
+      <div className="Account__flex Account__flex--center">
+        <p>Username:</p>
+        {isUsernameSaved ? (
```

Comment:
Could you add a button to change usernames as well if there is a saved username. It would re-open the form if the button to change usernames is pushed.